sites/public/src/pages/account/application/[id]/lottery-results.tsx

```diff
@@ -0,0 +1,169 @@
+
+  const preferenceRank = (rank: number, preferenceName: string, numApplicants: number) => {
+    return (
+      <Card.Section divider={"flush"} className={styles["preference-rank"]}>
+        <div className={styles["rank-number"]}>{`#${rank}`}</div>
+        <div>
+          <Heading priority={4} size={"lg"}>
+            {preferenceName}
+          </Heading>
+          <p className={styles["number-applicants"]}>
+            {t("account.application.lottery.applicantList", { applicants: numApplicants })}
+          </p>
+        </div>
+      </Card.Section>
+    )
+  }
+
+  return (
+    <>
+      <RequireLogin signInPath="/sign-in" signInMessage={t("t.loginIsRequired")}>
+        <FormsLayout className={styles["lottery-results"]}>
+          {noApplication && (
+            <ApplicationError error={t("account.application.noApplicationError")} />
+          )}
+          {unauthorized && <ApplicationError error={t("account.application.noAccessError")} />}
+          {application && (
+            <>
+              <ApplicationListingCard listingName={listing?.name} listingId={listing?.id} />
+              <BloomCard className={"mb-6"}>
+                <>
+                  <Card.Section divider={"inset"} className={"border-none"}>
+                    <Button
+                      size="sm"
+                      leadIcon={<Icon>{CustomIconMap.chevronLeft}</Icon>}
+                      variant={"text"}
+                      href={"/account/applications"}
+                    >
+                      {t("t.back")}
+                    </Button>
+                    <Heading priority={2} size={"2xl"} className="mt-6">
+                      {t("account.application.lottery.resultsHeader")}
+                    </Heading>
+                    <p className="mt-4">
+                      {t("account.application.lottery.resultsSubheader", {
+                        applications: 2500,
+                        units: 50,
+                      })}
+                      {/* TODO: Plug in BE data */}
+                    </p>
+                  </Card.Section>
+                  <Card.Section
+                    divider={"flush"}
+                    className={`${styles["background-card-section"]} border-none`}
+                  >
+                    <Heading priority={3} size={"xl"}>
+                      {t("account.application.lottery.rawRankHeader")}
+                    </Heading>
+                    <p className={styles["raw-rank"]}>57</p> {/* TODO: Plug in BE data */}
+                  </Card.Section>
+                  <Card.Section divider={"flush"}>
+                    <div>
+                      <p>{t("account.application.lottery.rawRank")}</p>
+                    </div>
+                    <div>
+                      <Button
+                        className={styles["section-button"]}
+                        href={"https://www.exygy.com"} // NOTE: Update per jurisdiction
```

Comment:
Would this be a good time to update these urls to the Doorway ones? Or is that going to be tackled when we have the data?